Section_prices/style.css

```diff
@@ -0,0 +1,136 @@
+/* font-family: 'Poppins', sans-serif;
+font-family: 'Roboto', sans-serif; */
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+.pricesWrap {
+    height: 100vh;
+    width: clac(1920px - 20px);
+    margin: 0 auto;
+}
+
+main {
+    margin: 0 auto;
+    height: 716px;
+    /* width: 1001px; */
+    width: 1133px
+}
+
+/* header text */
+
+.choosePlan {
+    width: 613px;
+    height: 141px;
+    margin: 0 auto;
+}
+
+.choosePlan h1 {
+    text-align: center;
+    font-family: 'Poppins', sans-serif;
+    font-size: 32px;
+    margin-bottom: 15px;
+    margin-top: 151px;
+    color: #3C374B;
+    font-weight: 600;
+}
+
+span.blue {
+    color: #2339C4;
+}
+
+.choosePlan p {
+    font-family: 'Roboto', sans-serif;
+    font-size: 16px;
+    text-align: center;
+    color: #505A78;
+}
+
+/* three column */
+
+.container {
+    display: flex;
+    margin-top: 117px;
+    margin-bottom: 322px;
+}
+
+.price {
+    width: 291px;
+    height: 458px;
+    box-shadow: 0 2px 10px rgba(0, 0, 0, 8%);
+}
+
+.price:nth-of-type(1) {
+    margin: 0 65px;
+    /* margin-right: 130px; */
+}
+
+.price:nth-of-type(2) {
+    margin-right: 65px;
+    /* margin-right: 130px; */
+
+}
```

Comment:
ww komentarz poazwala Ci usunąć te style:
* .price:nth-of-type(1)
* .price:nth-of-type(2)

Flex jest naprawde ciekawym narzędziem i warto z niego korzystać w pełnej okazałości